modules/installation-ibm-cloud-iam-policies-api-key.adoc

```diff
@@ -41,6 +41,12 @@ You must assign the required access policies to your IBM Cloud account.
 |Editor, Operator, Viewer, Administrator
 |
 
+|Account management ^[2]^
```

Comment:
This new policy is required when using an existing ResourceGroup and when IPI has to create a new ResourceGroup.
This requirement is necessary for IPI to lookup ResourceGroups during `manifest generation`, for `ccoctl` to create Secrets via CredentialsRequests, and other instances during cluster creation.

I recommend remove `[2]` from this policy, as it is always required (not optional).

I added some details to the Jira
https://issues.redhat.com/browse/OCPBUGS-11224?focusedId=22775393

Reply:
> I recommend remove `[2]` from this policy, as it is always required (not optional).

@cjschaef Thank you for the review. Removed the footnote. Please take one more look.